static/js/containers/SubscriptionsSidebar.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from "react"
+import { connect } from "react-redux"
+import { Link } from "react-router-dom"
+import type { Dispatch } from "redux"
+
+import { channelURL } from "../lib/url"
+import { actions } from "../actions"
+import type { RestState } from "../flow/restTypes"
+import type { Channel } from "../flow/discussionTypes"
+
+type SubscriptionsSidebarProps = {
+  subscriptions: RestState<Array<Channel>>,
+  dispatch: Dispatch
+}
+
+class SubscriptionsSidebar extends React.Component {
+  props: SubscriptionsSidebarProps
+
+  componentWillMount() {
+    const { subscriptions, dispatch } = this.props
+    if (!subscriptions.loaded && !subscriptions.processing) {
+      dispatch(actions.subscriptions.get())
+    }
+  }
+
+  render() {
+    const { subscriptions } = this.props
+
+    if (!subscriptions.data) {
+      return null
+    }
+
+    return (
+      <div className="subscriptions">
+        {subscriptions.data.map(channel =>
+          <div key={channel.name}>
+            <Link to={channelURL(channel.name)}>
+              {channel.title}
+            </Link>
+          </div>
+        )}
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    channels:      state.channels,
+    subscriptions: state.subscriptions
+  }
+}
+
+export default connect(mapStateToProps)(SubscriptionsSidebar)
```

Comment:
OK that sounds reasonable. I think I'm comfortable with the individual API calls to channels and channel detail for now, and if we decide on something cleaner than `SubscriptionsSidebar` as a container, I think that would be pretty low effort. 

I do think we should change the `channels` naming convention, though. `subscriptions` refers to an API call for multiple channels, and `channels` refers to an API call for a single channel. I really think we should change that and other similarly named variables. Also f we want to keep the notion of subscriptions in the variable names, `subscribedChannels` might be more helpful than `subscriptions`